Assets/Scripts/CSharp7/CSharp7.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class CSharp7 : MonoBehaviour
+{
+    [SerializeField] private int _skipCount;
+    [SerializeField] private int _takeCount;
+
+    private int _vectorCount = 5;
+    private List<int> xPositions;
+
+    public void Start()
+    {
+        var list = new List<Vector3>();
+        GetXPositions(list);
+        GetArray();
+    }
+
+    public List<int> GetXPositions(List<Vector3> vectors)
+    {
+        xPositions = vectors
+          .Where(vector => vector.x > _vectorCount)
+          .Select(vector => (int)vector.x)
+          .ToList();
+        return xPositions;
+    }
+
+    private void GetArray()
+    {
+        // Exercise 3.Skip, ToArray
+
+        var array = xPositions.ToArray()
```

Comment:
Check what type will be in var array
You need to get int[]

Reply:
+?